app/layout/aside_feed.phtml

```diff
@@ -8,8 +8,12 @@
 	}
 
 	$state_filter = FreshRSS_Context::$user_conf->show_fav_unread ? '&state=3' : '';
-?>
 
+	$urlParamState= Minz_Request::param('state','');
```

Comment:
nit

As an aside, it stands out a bit odd to me when the others are named things like $actual_view, $state_filter, etc.